kubeflow/trainer/rhai/traininghub.py

```
@@ -0,0 +1,199 @@
+        "        traceback.print_exc()",
+        "",
+        "    print('[PY] Training finished successfully.')",
+        "",
+    ]
+
+    if func_args is None:
+        call_line = "training_func({})"
+    elif isinstance(func_args, dict):
+        params_lines: list[str] = ["training_func({"]
+        for key, value in func_args.items():
+            params_lines.append(f"    {repr(key)}: {repr(value)},")
```

Ruff (0.14.4)

Use explicit conversion flag

Replace with conversion flag

(RUF010)